csc302-spring-2021/proj-DaTeam
# refactor: move page routing out of Nav component and use useLocation hook to check path

## Description



- The Nav component shouldn't be concerned with rendering pages, it should just render the navbar itself
- Swapped out history.push() calls for Link component
- Made responses the default route

## Checklist


## QA Steps



1. `cd frontend`
1. `npm start`
1. Verify that frontend looks visually the same as before
1. Verify that you can navigate between the pages by clicking the links in the navbar
1. Verify that going to `/` route redirects you to `/responses`
